src/main/java/nl/aerius/print/ExportJob.java

```diff
@@ -94,6 +95,12 @@ public ExportJob chromeHost(final String host) {
     return this;
   }
 
+  public ExportJob retry(final Integer retryCount) {
```

Comment:
Use a primitive int (as that is also the parameter of the chrome retry method?
```suggestion
  public ExportJob retry(final int retryCount) {
```

Reply:
It is actually useful to have a nullable type here, because the chromedriver uses the default value (3) when retry is `null`

Reply:
But `chrome.retry` has an `int` as parameter?

Reply:
oh right, that's dumb. I'll make sure it is initialized with the default value then.